__tests__/app.test.js

```diff
@@ -52,68 +52,106 @@ describe('getTopics', () => {
-      .get('/api/articles/ABD')
-      .expect(400)
-      .then((result) => {
-        expect(result.body.msg).toBe('Bad request, invalid article_id');
-      })
+    .get('/api/articles/ABD')
+    .expect(400)
+    .then((result) => {
+      expect(result.body.msg).toBe('Bad request, invalid article_id');
     })
   })
+})
 describe('getUsers', () => {
   test('200: /api/users responds with an array', () => {
     return request(app)
-      .get('/api/users')
-      .expect(200)
-      .then((res) => {
-        expect(res.body).toBeInstanceOf(Array);
-      });
+    .get('/api/users')
+    .expect(200)
+    .then((res) => {
+      expect(res.body).toBeInstanceOf(Array);
+    });
   });
   test('200 /api/topics responds with an array of users, formatted correctly', () => {
     return request(app)
-      .get('/api/users')
-      .expect(200)
-      .then((res) => {
-        expect(res.body).toBeInstanceOf(Array);
-        expect(
-          res.body.forEach((user) => {
-            expect(user).toMatchObject({
-              username: expect.any(String),
-            });
-          })
+    .get('/api/users')
+    .expect(200)
+    .then((res) => {
+      expect(res.body).toBeInstanceOf(Array);
+      expect(
+        res.body.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+          });
+        })
         );
       });
-  });
-  test('/api/userr responds 404 not found', () => {
-    return request(app)
+    });
+    test('/api/userr responds 404 not found', () => {
+      return request(app)
       .get('/api/userr')
       .expect(404)
       .then((result) => {
         expect(result.body.msg).toBe('invalid path');
       });
+    });
   });
-});
+  
+  describe('getArticles', () => {
+    test('200 GET /api/articles responds with an array of article objects', () => {
+      return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((result) => {
+        expect(result.body).toBeInstanceOf(Array)
+        expect(result.body.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number)
+          })
+        }))
+        expect(Number(result.body[5].comment_count)).toBe(11)
+      })
+    })
+    test('200 GET /api/articles sorts articles by date', () => {
+      return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((result) => {
+        expect(result.body).toBeSortedBy('created_at', {descending: true})
+      })
+      })
+      test('/api/article responds 404 not found', () => {
+        return request(app)
+          .get('/api/article')
+          .expect(404)
+          .then((result) => {
+            expect(result.body.msg).toBe('invalid path');
+          });
+      });
```

Comment:
For now, this does seem like the only thing that can wrong on this endpoint, but I would say you've already handled it previously! The 404 you did for `/api/topic` is a generic "this an incorrect path/typo/spelling mistake" scenario where we only need to consider it once. Just the one test for this error will do it - no need to consider lots of typos!